src/main/scala/org/pantsbuild/jarjar/JJProcessor.scala

```diff
@@ -0,0 +1,18 @@
+package org.pantsbuild.jarjar
```

Comment:
I'm afraid we can't, since process method in JarProcessor is package private.
I'm not sure why they use such a method signature.